go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
 	github.com/hashicorp/go-hclog v1.6.2
 	github.com/hashicorp/go-plugin v1.6.0
+	github.com/jaegertracing/jaeger/model v0.0.0-00010101000000-000000000000
```

Comment:
What is the significance of this weird version? Is dependabot going to try to change it? Did you try on a separate repo?

Reply:
I will try in my repository and see what happens